File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_project_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusProjectVO:
    """项目信息VO对象(维护项目基本信息、状态及关联关系)"""
    created_at: str = None
    updated_at: str = None
    id: int = None
    uuid: str = None
    name: str = None
    code: str = None
    description: str = None
    status: str = None
    leader_id: int = None
    start_time: str = None
    end_time: str = None
    budget_amount: int = None
    is_deleted: bool = None

File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/PlusProjectVO.kt
package com.sdkwork.backend

data class PlusProjectVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val id: Int? = null,
    val uuid: String? = null,
    val name: String? = null,
    val code: String? = null,
    val description: String? = null,
    val status: String? = null,
    val leaderId: Int? = null,
    val startTime: String? = null,
    val endTime: String? = null,
    val budgetAmount: Int? = null,
    val isDeleted: Boolean? = null
)
